### ISSUE TITLE
RFE: Improve netapp.log logging format

### ISSUE THREAD

**Issue by dutchiechris** [NetApp/netappdvp#61]
As of v1.3 logging with `debug: false` in the conf file leads to log entries that look like this:

```
INFO[0000] Using storage driver: solidfire-san
INFO[0000] Using config: {1 solidfire-san false false [] []}
INFO[0000] Starting docker volume plugin with the following options:  port=27609 volume-driver=netapp volumeDir=/var/lib/docker-volumes/netapp
```
and with `debug:true` you get this:
```
time="2017-01-25T13:58:57Z" level=info msg="Using storage driver: solidfire-san"
time="2017-01-25T13:58:57Z" level=info msg="Using config: {1 solidfire-san true false [] []}"
time="2017-01-25T13:58:57Z" level=info msg="Starting docker volume plugin with the following options:" port=27609 volume-driver=netapp volumeDir="/var/lib/docker-volumes/netapp"
```

Request is:
1. Use the same logging format regardless of debug level
1. Ensure entries have a timestamp regardless of debug level

**Comment by clintonk** [comment]
Thanks for the suggestion, Chris.  We already have a user story to improve nDVP logging, and I've captured this request there.

**Comment by adkerr** [comment]
Internally tracked by [US2168](https://rally1.rallydev.com/#/47479883219d/detail/userstory/86687827952)

**Comment by adkerr** [comment]
Fixed in bbcb41d7ce96a4170e24ccccb2986c5423ab27f4
